regexes/device/mobiles.yml

```diff
@@ -1700,6 +1720,10 @@ Explay:
     - regex: 'Explay[ _-]([^;/]+)[;/)]'
       model: '$1'
 
+    - regex: '(RioPlay)[;/) ]'
+      device: 'smartphone'
+      model: '$1'
```

Comment:
When detecting single devices it's not required to use `$1`. Instead of this it's better to directly set the model. In this case the device model would be `Rio Play`.